parser/proto.y

```diff
@@ -49,7 +45,7 @@ import (
 	mtdMsgType   *ast.RPCTypeNode
 	mtdElements  []ast.RPCElement
 	opt          *ast.OptionNode
-	optN         nodeWithEmptyDecls[*ast.OptionNode]
+	optWE        nodeWithEmptyDecls[*ast.OptionNode]
```

Comment:
Is "WE" (with empty?) the new notation for "N"? If so, should `msgFld` and `msgGrp` above instead be `fldWE` and `grpWE`?

Reply:
I changed the opt->optRaw instead (without empty decls is the corner case)